silkrpc/core/evm_tracing.cpp

```diff
@@ -79,6 +79,15 @@ void to_json(nlohmann::json& json, const Trace& trace) {
     }
 }
 
+void to_json(nlohmann::json& json, const std::vector<Trace>& traces) {
+    json = nlohmann::json::array();
+    for (auto& trace : traces) {
+        nlohmann::json entry;
+        to_json(entry, trace);
+        json.push_back(entry);
+    }
+}
```

Comment:
It should be sufficient to define serialization for `Trace` here (i.e. `void to_json(nlohmann::json& json, const Trace& trace)`) because Nlohmann JSON library knows how to handle sequences of serializable types.